getting_started/setup.adoc

```diff
@@ -0,0 +1,148 @@
+= Setup
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== System Requirements
+
+OpenShift components can be installed across multiple hosts. For Beta 1, you can deploy a link:../architecture/kubernetes_infrastructure.html#master[master] on one host, and two link:../architecture/kubernetes_infrastructure.html#node[nodes] on two separate hosts.
+
+The following sections describe the system requirements per component type.
+
+=== Master
+
+Masters must meet the following system requirements:
+
+- Physical or virtual system
+ifdef::openshift-origin[]
+- Base OS: Fedora 21, CentOS 7, or Red Hat Enterprise Linux Server 7.0
+endif::[]
+ifdef::openshift-enterprise[]
+- Base OS: Red Hat Enterprise Linux Server 7.0
+endif::[]
+- 2 vCPU
+- Minimum 8 GB RAM
+- Minimum 30 GB hard disk space
+
+=== Nodes
+
+Nodes must meet the following system requirements:
+
+- Physical or virtual system, or an instance running on a public IaaS
+ifdef::openshift-origin[]
+- Base OS: Fedora 21, CentOS 7, or Red Hat Enterprise Linux Server 7.0
+endif::[]
+ifdef::openshift-enterprise[]
+- Base OS: Red Hat Enterprise Linux Server 7.0
+endif::[]
+- 1 vCPU
+- Minimum 8 GB RAM
+- Minimum 15 GB hard disk space
+
+
+=== Other Beta Requirements
+
+- You must have a GitHub account or a local (on premise) Git server.
+- A shared network must exist between the master and node components.
+
+== Prerequisites
+
+Whether installing OpenShift using RPM packages or Docker images, you must first satisfy the following prerequisites.
+
+ifdef::openshift-enterprise[]
+=== Installing Red Hat Enterprise Linux 7
+
+As mentioned in the system requirements, a base installation of Red Hat Enterprise Linux (RHEL) 7.0 is required for master or node hosts. More information on installing RHEL 7.0 can be found in the following documentation:
+
+https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Installation_Guide/index.html[Red Hat Enterprise Linux 7 Installation Guide]
+
```

Comment:
Probably best to stick to our general convention when referring to other docs; for example:

As mentioned in the system requirements, a base installation of Red Hat Enterprise Linux (RHEL) 7.0 is required for master or node hosts. See the https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Installation_Guide/index.html[Red Hat Enterprise Linux 7 Installation Guide] for more information on installing RHEL 7.0.